makefile

```diff
@@ -106,7 +106,7 @@ validate: clean-dist install tsc lint stylelint test validate-build
 	$(call log, "everything seems 👌")
 
 validate-prepush:
-	@run-p tsc lint-staged "test -- --verbose  --runInBand --onlyChanged"
```

Comment:
The secret here is that lint-staged is a bit useless on push as nothing is staged at that point, it's designed to be used on commit hook. It's on my todo list to fix 🤦

Reply:
ah maybe that's why i saw it - i must have had something staged but not committed while trying to push

Reply:
shall i close it? is it not much use?